tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -375,3 +375,35 @@ def test_consecutive_fingerprint_consistency(cpu_template_helper, tmp_path):
 
     # Compare them.
     cpu_template_helper.fingerprint_compare(fp1, fp2, None)
+    # Strip common entries.
```

Comment:
Are these changes related to live migration/msync or are they backports from another branch?

Reply:
Backports from main actually